extension/src/react/components/App.jsx

```diff
@@ -29,11 +32,20 @@ const App = () => {
             <Card 
                 title="Github Guru" 
                 className='base' 
-                tabList={tabs}
-                activeTabKey={activeTab}
-                onTabChange={key => setActiveTab(key)}
             >
-                {content[activeTab]}
+                <Space direction="vertical" size="middle"> 
+                    <SegmentedComponent tabs={tabs} onChange={setActiveTab} />
+                    {activeTab === "Notifs" && (
+                        <>
+                            <SelectRepo />
+                            <PrCard />
+                            <PrCard />
+                            <PrCard />
+                            <PrCard />
+                        </>
```

Comment:
Rendering this in app.jsx like this doesnt make any sense. Each tab is going to look very different so once we are done, if we were to render each tab like this, we would have this super long convoluted file. Its better if we go the route of creating a component for each: Notifications, Pull Requests, and Analytics. In those components we can render the select menu and all the cards, but doing this here is going to get very complicated very fast.